crates/db_schema/src/impls/community.rs

```diff
@@ -236,6 +237,21 @@ impl Bannable for CommunityPersonBan {
   }
 }
 
+impl CommunityFollower {
+  pub fn to_subscribed_type(follower: &Option<Self>) -> SubscribedType {
+    match follower {
+      Some(f) => {
+        if f.pending {
+          SubscribedType::Pending
+        } else {
+          SubscribedType::Subscribed
+        }
+      }
+      None => SubscribedType::NotSubscribed,
+    }
+  }
+}
```

Comment:
Hmm not sure about this one. How about using a database `column community_follower.status`, with values either 0, 1, 2? Though that would overcomplicate things in a different way. There should at least be a comment here to explain how it works.

Reply:
There is no row if its not subscribed, so only the pending column needs to exist there.